circle/test.env

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY_BASE=$SECRET_KEY_BASE
+OMNIAUTH_CLIENT_SECRET=$OMNIAUTH_CLIENT_SECRET
+OMNIAUTH_CLIENT_ID=$OMNIAUTH_CLIENT_ID
+DEVISE_SECRET_KEY=$DEVISE_SECRET_KEY
+AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+AWS_REGION=$AWS_REGION
+SQS_QUEUE_URL=$SQS_QUEUE_URL
```

Comment:
What does `test.env` do? Is it setting environment variables just for testing, using different variables for deployment?